### ISSUE TITLE
Configure initial style-dictionary

### ISSUE THREAD

**Issue by viktorrusakov** [openedx/paragon#1543]
**Can be done only after #1542**

After successfully categorizing SCSS variables into design tokens we are ready to build initial configuration of style dictionary.

**AC**
- create all necessary JSON files according to the schema defined in #1542
- Configure style-dictionary to include the following platforms
   - CSS variables 
   - SCSS variables 
   - **Nice to have** CSS utility classes (may need a custom formatter)

**Notes**
- you can use https://github.com/openedx/paragon/tree/ags/style-dictionary-poc branch as a baseline for you PR
- useful link about style dictionaries
    - https://github.com/amzn/style-dictionary
    - https://amzn.github.io/style-dictionary/#/architecture
